main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
+Issuers are a one-to-one relationship with both a [mint](#mint) and a [brand](#brand), so each issuer works
 with one and only one asset type, such as only working with quatloos or only working
-with moola. They can create empty [purses](#purse) and [payments](#payment) for
-their asset type, but cannot mint new [amounts](#amounts). 
+with moola. This association cannot change to another type. 
 
-Issuers can also transform
+Issuers can create empty [purses](#purse) and [payments](#payment) for
+their asset type, but cannot mint new [amounts](#amount). Issuers can also transform
 payments of their asset type (splitting, combining, burning, and exclusively claiming
 payments). An issuer from a trusted source can determine if an untrusted payment of
 its asset type is valid. 
 
-See the [ERTP Guide's Issuer section](/ertp/guide/issuers-and-mints.md)
-and the [ERTP API's Issuer section](/ertp/api/issuer.md) for more information.
+For more information, see the [ERTP Guide's Issuer section](/ertp/guide/issuers-and-mints.md)
+and the [ERTP API's Issuer section](/ertp/api/issuer.md).
+
+## Keywords
+
+Keywords are unique identifiers per contract. They tie together the [proposal](#proposal), [payments](#payment) 
+to be [escrowed](#escrow), and [payouts](#payout) to the user by serving as keys for key-value pairs in various 
+records with values of [amounts](#amount), [issuers](#issuer), etc.
 
 ## Mint
-In ERTP, mints create digital assets and are the only objects with the authority to do so. 
-Access to a mint gives you the power to create more digital assets of its type at will. Mints
-can only create one type of asset. 
+Agoric has two mint objects, *ERTP mints* and *Zoe Contract Facet mints (ZCFMints)*. They both create
+digital assets. Which mint type creates an asset doesn't matter; quatloos created by an ERTP mint are 
+indistinguishable from quatloos created by a ZCFMint. 
 
-Mints are [issuer's](#issuer) admin facets, and there is a one-to-one relationship between an issuer and
-its mint.
+- ERTP mints create digital assets asynchronously and are the only ERTP objects with the authority to do so. 
+  Access to an ERTP mint gives you the power to create more digital assets of its type at will. Mints
+  can only create one type of asset and cannot change to create a different type.
 
-See the [ERTP Guide's Mint section](/ertp/guide/issuers-and-mints.md#mints) 
-and the [ERTP API's Mint section](/ertp/api/mint.md) for more information.
+  ERTP mints are [issuer's](#issuer) admin [facets](#facet), and there is a one-to-one relationship between an issuer and
+  its mint. ERTP mints are also in a one-to-one relationship with that issuer's associated [brand](#brand).
+
+- ZCFMints are synchronous Zoe mints, which mint and reallocate digital assets synchronously 
+  instead of asynchronously like ERTP mints. Similar to ERTP mints, they have one-to-one relationships
+  with an issuer and its associated brand.
```

Comment:
No, first, objects are not synchronous or asynchronous. Second, the `mintPayment` method on the underlying ERTP mint is synchronous, so that is not the distinction. The distinction is that ZCFMints make it much easier to mint and escrow the payments to a particular seat.